src/ocm/components/clusterWizard/ClusterDetailsForm.tsx

```diff
@@ -136,6 +137,7 @@ const ClusterDetailsForm = (props: ClusterDetailsFormProps) => {
                   managedDomains={managedDomains}
                   clusterExists={!!cluster}
                   clusterCpuArchitecture={cluster?.cpuArchitecture}
+                  canSelectCpuArch={cluster ? canSelectCpuArchitecture(cluster) : true}
```

Comment:
It's not possible to change the CPU architecture after a cluster is created, so when this needs to be determined is when the cluster is empty, in which case this function returns always true.

The function `canSelectCpuArchitecture` would only work for this case in Day2, not in Day1.

Reply:
Done changes in uhc-portal to day1: https://gitlab.cee.redhat.com/service/uhc-portal/-/merge_requests/3967